azure/services/resourceskus/client.go

```diff
@@ -57,7 +57,7 @@ func (ac *AzureClient) List(ctx context.Context, filter string) ([]compute.Resou
 	ctx, _, done := tele.StartSpanWithLogger(ctx, "resourceskus.AzureClient.List")
 	defer done()
 
-	iter, err := ac.skus.ListComplete(ctx, filter)
+	iter, err := ac.skus.ListComplete(ctx, filter, "true")
```

Comment:
Would be good to doble check if it's fine to put `"true"` in here.

Reply:
Can be resolved, will be discussed in https://github.com/kubernetes-sigs/cluster-api-provider-azure/pull/2277.